src/buspirate.c

```diff
@@ -63,6 +63,8 @@
 #define BP_FLAG_XPARM_CPUFREQ       (1<<5)
 #define BP_FLAG_XPARM_RAWFREQ       (1<<6)
 #define BP_FLAG_NOPAGEDREAD         (1<<7)
+#define BP_FLAG_PUPS                (1<<8)
+#define BP_FLAG_HIZ                 (1<<9)
```

Comment:
Likewise, `BP_FLAG_HI_Z` would immediately explain to me the purpose of the bit.